cg/cli/backup.py

```diff
@@ -18,25 +18,30 @@ def backup(context: click.Context):
 
 @backup.command("fetch-flowcell")
 @click.option("-f", "--flowcell", help="Retreive a specific flowcell")
-@click.option("--dry", is_flag=True, help="Only log, do not contact PDC")
+@click.option(
+    "--dry-run", "dry_run", is_flag=True, help="Don't retrieve from PDC or set flowcell's status"
+)
 @click.pass_context
-def fetch_flowcell(context: click.Context, dry: bool, flowcell: str):
+def fetch_flowcell(context: click.Context, dry_run: bool, flowcell: str):
     """Fetch the first flowcell in the requested queue from backup."""
     status_api = Store(context.obj["database"])
+    max_flowcells = context.obj.get("max_flowcells", 1250)
```

Comment:
Put the default in a global variable? Perhaps with a comment about the choice of 1250
```suggestion
    max_flowcells = context.obj.get("max_flowcells", MAX_FLOWCELLS)
```